src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -648,8 +648,8 @@ uct_tcp_sockcm_ep_server_invoke_conn_req_cb(uct_tcp_sockcm_ep_t *cep)
     ucs_debug("fd %d: remote_data: (field_mask=%"PRIu64") "
               "dev_addr: %s (length=%zu), conn_priv_data_length=%zu",
               cep->fd, remote_data.field_mask,
-              ucs_sockaddr_str((const struct sockaddr*)remote_data.dev_addr,
-                               peer_str, UCS_SOCKADDR_STRING_LEN),
+              ucs_sockaddr_str((const struct sockaddr*)&saddr, peer_str,
```

Comment:
minor: maybe use `client_saddr.addr` just because it's already casted

Reply:
done